src/blocks/recommendations/block.json

```diff
@@ -0,0 +1,44 @@
+{
+	"$schema": "https://schemas.wp.org/wp/5.9/block.json",
+	"apiVersion": 2,
+	"name": "wp-parsely/recommendations",
+	"title": "Parse.ly Recommendations",
```

Comment:
As per https://core.trac.wordpress.org/changeset/49981 then `title`, `description`, each `keyword`, and each style `label` are automatically wrapped in `translate_with_gettext_context()` with the textdomain passed from the value on line 7 ✅